crates/tracing/src/lib.rs

```diff
@@ -58,6 +58,13 @@ pub fn configure_observability(
 
     if otel_config.metrics_enabled() {
         metrics::configure_metrics(&normalized_service_name, otel_config)?;
+
+        // Create and register the error counters
+        let meter = global::meter("wasmcloud");
+        let error_counter = meter
+            .u64_counter("wasmcloud_errors")
+            .with_description("Counts the number of errors in wasmcloud")
+            .init();
```

Comment:
I would suggest sticking this code inside of `configure_metrics`, or where we set up other counters. It also might be better to call this `wasmcloud_dropped_logs`